spec/02-integration/05-proxy/05-dns_spec.lua

```diff
@@ -145,5 +145,60 @@ for _, strategy in helpers.each_strategy() do
         assert.response(r).has.status(503)
       end)
     end)
+    describe("udp sockets", function()
```

Comment:
Of course, we should write tests for all fixes, but this issue is caused by third-party dependencies and I'm not sure how we should do it.

I think it's too heavy for introducing an integration test for this purpose, I'd prefer to use unit tests to cover it.

The perfect thing to do would be to have `lua-resty-dns` cover the issue, but until then I think it would be better to use unit tests instead.

Reply:
I suspect that even if we change this pr to a patch, the test case is still coverable.

Reply:
Yes, the issue can be covered, but does it make more sense to move the tests to unit tests?

Reply:
I'm not sure that makes more sense, just a suggestion.

Reply:
This comment is not a blocker for this PR.